app/bundles/CoreBundle/Monolog/Handler/PhpHandler.php

```diff
@@ -1,72 +0,0 @@
-<?php
```

Comment:
Why was this removed? Unless monolog has changed upstream to write die statements at the beginning of the log, removing this will expose Mautic's logs to the web.

Reply:
Please look here: https://github.com/mautic/mautic/pull/6599#discussion-diff-223220882L816

Reply:
I'm having a hard time finding that discussion. The link takes me to a place that doesn't seem to be the same context. I tried searching the code diff for discussions and couldn't find anything for this context.

Reply:
This change is also related: https://github.com/mautic/mautic/pull/6848/files#diff-966ddd047b8bdaf985231305e091f415L816

Why this class was removed:
1. Monolog 2.x previously used constants like `monolog.handler.stream.class` and any bundle was able to redefine it.
2. There was some issues with Monolog 2.x, and it was upgraded to 3.x
3. Monolog 3.x is not supports constant `monolog.handler.stream.class` anymore, and this class cannot be redefined.
4. But actually this file wasn't used since this PR: https://github.com/mautic/mautic/pull/721 when Logger's `type` was changed from `stream` to `rotating_file`, so we can just remove it.

Reply:
Thanks for that! Hmmm, then that's been an issue for a long time and not good. Will deal with it outside this PR then.